README.md

```diff
@@ -38,7 +38,16 @@ localhost:8080 and get a response from the server.
 Android application
 -------------------
 
-TODO
+The Android application has one activity, which retrieves the `'requested'` key
+of the JSON object retrieved from the URL http://10.0.0.2:8080/ and displays it
```

Comment:
I've fixed it to match the URL actually used in the app, but localhost is the phone, even if the phone is an emulator. 10.0.2.2 is the host device for an emulator, and thus what is used here.